MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,127 @@
+#  Accessibility Support in Maui Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selection using keyboard interactions.
+
+#### Day, Week and WorkWeek views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the same time slot on the next day.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the same time slot on the previous day.<br/><br/></td></tr>
+<tr>
+<td>
+Down arrow<br/><br/></td><td>
+Moves selection to the next time slot directly below the currently selected time slot.<br/><br/></td></tr>
+<tr>
+<td>
+Up arrow<br/><br/></td><td>
+Moves selection to the previous time slot directly above the currently selected time slot.<br/><br/></td></tr>
+</table>
+
+#### Timeline views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the next time slot of the currently selected time slot.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the previous time slot of the currently selected time slot.<br/><br/></td></tr>
+</table>
+
+#### Month view
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the next date of the currently selected date.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the previous date of the currently selected date.<br/><br/></td></tr>
+<tr>
+<td>
+Down arrow<br/><br/></td><td>
+Moves selection to the date directly below the currently selected date on the next row.<br/><br/></td></tr>
+<tr>
+<td>
+Up arrow<br/><br/></td><td>
+Moves selection to the date directly above the currently selected date on the previous row.<br/><br/></td></tr>
+</table>
+
+#### Appointments
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Moves selection to the next appointment of the currently selected appointment.<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Moves selection to the previous appointment of the currently selected appointment.<br/><br/></td></tr>
+<tr>
+<td>
+Delete<br/><br/></td><td>Deletes the selected appointment from appointments collection.<br/><br/></td></tr>
+</table>
+
+#### View navigations
+
+<table>
+<tr>
+<td>
+Ctrl + Alt + 1 => DayView<br/><br/></td><td>
+Moves the view to day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 2 => WeekView<br/><br/></td><td>
+Moves the view to week view.<br/><br/></td></tr>
```

Comment:
Add MAC platform keys for view navigation

Reply:
Added MAC platform keys for view navigation.